### DISCUSSION TITLE
How to get select text in LineEdit?

### DISCUSSION THREAD

**Discussion by felixmaker** [slint-ui/slint#3899] [Q&A]
If I don't want to use copy or cut method...

**Comment by felixmaker** [comment]
What I'm trying to do is:

1. select some text in LineEdit
2. clicking a button than I get the text
3. do something and returning some new text 
4. replace the selection by the new text

**Comment by ogoffart** [comment]
This API is missing at this point. I created https://github.com/slint-ui/slint/issues/3949  to track this feature request.
